helm/oauth2-proxy/Chart.yaml

```
@@ -1,7 +1,7 @@
 name: oauth2-proxy
-version: 7.9.2
+version: 7.9.3
```

```suggestion
version: 7.10.0
```

it's not a patch really should be a minor bump instead